### PR TITLE
Add standalone main

### PR DESCRIPTION
**Pull request by hauntsaninja**

### CODE REVIEW

**Comment by hauntsaninja** [comment]
I use similar code in https://github.com/hauntsaninja/no_implicit_optional and it's fine

**Comment by JelleZijlstra** [comment]
CI is failing because of pyanalyze, at least some of it should be fixed in a later version. I'll make a separate PR upgrading some CI stuff.

File: README.md [hunk 1]
```diff
@@ -85,6 +79,17 @@ There are two shortcut flags to enable multiple transformations at once:
   `--int-param`, `--float-param`, `--str-param`, `--bytes-param`, and
   `--annotate-imprecise-magics`.
 
+# LibCST
+
+autotyping is built as a LibCST codemod; see the
```
Comment (JelleZijlstra) [inline README.md]:
```suggestion
Autotyping is built as a LibCST codemod; see the
```
Consistent with next section

**Review by JelleZijlstra — APPROVED** [review]
Thank you!

**Pull request merged**